### PR TITLE
feat(rollapp params): remove maxgas from rollappparams

### LINKED ISSUE #523: change blockmaxgas from uint32 to int64

### LINKED ISSUE #525: remove blockmaxgas from rollappparams
maxgas param from cosmos params module will be used instead to avoid inconsistencies

### PR DESCRIPTION
**Pull request by srene**
## Description

----

Closes #525

**All** items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow-up issues.

PR review checkboxes:

I have...

- [ ]  Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ]  Targeted PR against the correct branch
- [ ]  included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ]  Linked to the GitHub issue with discussion and accepted design
- [ ]  Targets only one GitHub issue
- [ ]  Wrote unit and integration tests
- [ ]  Wrote relevant migration scripts if necessary
- [ ]  All CI checks have passed
- [ ]  Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code)
- [ ]  Add an issue in the [e2e-tests repo](https://github.com/dymensionxyz/e2e-tests) if necessary

SDK Checklist
- [ ] Import/Export Genesis
- [ ] Registered Invariants
- [ ] Registered Events
- [ ] Updated openapi.yaml
- [ ] No usage of go `map`
- [ ] No usage of `time.Now()`
- [ ] Used fixed point arithmetic and not float arithmetic
- [ ] Avoid panicking in Begin/End block as much as possible
- [ ] No unexpected math Overflow
- [ ] Used `sendCoin` and not `SendCoins`
- [ ] Out-of-block compute is bounded
- [ ] No serialized ID at the end of store keys
- [ ] UInt to byte conversion should use BigEndian

Full security checklist [here](https://www.faulttolerant.xyz/2024-01-16-cosmos-security-1/)

----

For Reviewer:

- [ ]  Confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ]  Reviewers assigned
- [ ]  Confirmed all author checklist items have been addressed

----

After reviewer approval:

- [ ]  In case the PR targets the main branch, PR should not be squash merge in order to keep meaningful git history.
- [ ]  In case the PR targets a release branch, PR must be rebased.

### CODE REVIEW

File: x/rollappparams/types/genesis_test.go [hunk 1]
```diff
@@ -45,33 +45,6 @@ func TestGenesisState(t *testing.T) {
...
-			name: "block max gas too small",
-			params: func() types.Params {
-				p := types.DefaultParams()
-				p.Blockmaxgas = 0
-				return p
-			},
-			expectedErr: true,
-		},
-		{
-			name: "block max size too small",
```
Comment (omritoptix) [inline x/rollappparams/types/genesis_test.go:58]: why did we remove the max size test?

File: x/rollappparams/types/params.go [hunk 2]
```diff
@@ -14,25 +14,20 @@ const (
 	DefaultDA = "mock"
 	// version used for the rollapp binary. it must be overwritten in Makefile.
 	DefaultVersion = "3a19edd887a9b576a866750bc9d480ada53d2c0d"
-	// default max gas accepted per block. limited to 400M.
-	DefaultBlockMaxGas = 400000000
-	// default max block size accepted (equivalent to block max size it can fit into a celestia blob).
-	DefaultBlockMaxSize = 500000
```
Comment (omritoptix) [inline x/rollappparams/types/params.go:20]: why did we remove the max size validation ?

File: x/rollappparams/types/genesis_test.go (hunk 1 above)
Comment (srene) [reply x/rollappparams/types/genesis_test.go:58]: no reason for that, it was by mistake. tests readded

File: x/rollappparams/keeper/keeper.go [hunk 3]
```diff
@@ -56,3 +46,8 @@ func (k Keeper) Version(ctx sdk.Context) (res string) {
 	k.paramSpace.Get(ctx, types.KeyVersion, &res)
 	return
 }
+
+func (k Keeper) BlockMaxSize(ctx sdk.Context) (res uint32) {
```
Comment (omritoptix) [inline x/rollappparams/keeper/keeper.go:50]: should we name it `BlockMaxBytes`?

Comment (srene) [reply x/rollappparams/keeper/keeper.go]: done

**Review by omritoptix — APPROVED** [review]

**Pull request merged**